antora.yml

```diff
@@ -1,5 +1,5 @@
 name: tinymce
-title: TinyMCE Documentation
+title: TinyMCE Documentation [BETA]
```

Comment:
This won't work, as this would mark all documentation as "beta" (e.g. for 5.x as well). You'll need to do it on the version instead or find another suitable variant.

Reply:
as this would mark all documentation as "beta"

how so? I could add [BETA 6.0]

Reply:
Antora groups things by this title/product name (take a look at staging down in the bottom right), so this would either end up showing as a separate product or would change all other products. So we need to change something that's going to be specific to 6.0 instead. I guess if you're happy with it showing as a separate product then this would be okay, but that seemed like a less than ideal solution.

Reply:
![image](https://user-images.githubusercontent.com/1575550/158273095-bab7d4eb-078c-4718-83de-7186c4b660d6.png)
Screenshot as that might be easier.

Reply:
Discussed this more in a zoom call this morning. For now we're only going to release the TinyMCE 6 docs and by the time we need to add others we should no longer need this. If we do then we'll re-evaluate then.